example/android/app/build.gradle

```diff
@@ -25,7 +25,7 @@ apply plugin: 'com.android.application'
 apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 
 android {
-    compileSdkVersion 32
+    compileSdkVersion 31
```

Comment:
Why did we make this change?

Reply:
We set to 31 for other plugins as well. It was mistyped for 32 earlier so change it back to 31 for consistency.

Reply:
It was always set to 31, there was a typo and changed to 32.  Now changed it back.